scripts/cfn

```diff
@@ -23,14 +23,32 @@ def upload_template_to_s3(conn, region, bucket_name, key_name, template):
         elif region == 'eu-west-1':
             region = boto.s3.connection.Location.EU
 
-        cloudformation_bucket = conn.create_bucket(bucket_name, location=region)
+        cloudformation_bucket = conn.create_bucket(bucket_name,
+                                                   location=region)
     key = cloudformation_bucket.new_key(key_name)
     key.set_contents_from_string(template)
     cfn_template_url = "https://s3.amazonaws.com/%s/%s" % (
         bucket_name, key_name)
     return cfn_template_url
 
 
+def validate_template(conn, template=None, url=None):
+    """Validate the template"""
+    print("Validating Template")
+    try:
+        if url:
+            conn.validate_template(template_url=url)
+        else:
+            conn.validate_template(template_body=template)
+    except boto.exception.BotoServerError as e:
+        # XXX - need to figure out why this isn't getting parsed from boto.
+        print("Error: %s" %
+              (literal_eval(e.error_message)['Error']['Message'],))
+        print("Exiting...")
+        sys.exit(1)
+    print("Template Validated")
```

Comment:
Anything that validates returns a `boto.cloudformation.template.Template` object.  If it does not validate it will throw an error:

```py
BotoServerError: BotoServerError: 400 Bad Request
<ErrorResponse xmlns="http://cloudformation.amazonaws.com/doc/2010-05-15/">
  <Error>
    <Type>Sender</Type>
    <Code>ValidationError</Code>
    <Message>Template format error: unsupported type or structure. (line 2, column 5)</Message>
  </Error>
  <RequestId>aacaaf5b-2442-11e6-96bc-4793db5d2621</RequestId>
</ErrorResponse>
```

The style of this script is that any error causes a fault.  We could cause this method to return `True` or `False`, but it steps away from the style of the other methods.  Your call, I'm happy to implement either.